crimsobot/models/fun_facts.py

```diff
@@ -0,0 +1,79 @@
+import random
+from typing import Any
+
+from tortoise import fields
+from tortoise.models import Model
+
+from crimsobot.models import DiscordUser
+from crimsobot.models.user import User
+
+
+class NoFactsExist(Exception):
+    def __init__(self, *args: Any):
+        if args:
+            self.message = args[0]
```

Comment:
`self.message = args[0] if args else None`?

Reply:
Technically yes, but there's no fault in being a little more explicit here I guess